package.json

```diff
@@ -75,7 +75,7 @@
     "@hmpps-book-secure-move-frameworks/1.1.0": "git+https://github.com/ministryofjustice/hmpps-book-secure-move-frameworks.git#v1.1.0",
```

Comment:
`ncu` throws an error for me when it's `git+https://` rather than `https://` but it obviously works ok for you

Reply:
Ahhh, I did fix this, then apparently running the command overwrote that. 

I think the reason it no longer complains though is because I added it to the `reject` list. I needed to do that regardless because otherwise it will try and update the previous versions to all point to the latest, which would break our backwards compatibility.